eng/helixpublish.proj

```diff
@@ -0,0 +1,122 @@
+<Project Sdk="Microsoft.DotNet.Helix.Sdk" DefaultTargets="Test">
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <IncludeDotNetCli>true</IncludeDotNetCli>
+    <DotNetCliPackageType>runtime</DotNetCliPackageType>
+    <!-- Set to match global.json.tools.dotnet by pre-build.ps1 -->
+    <DotNetCliVersion>$(MicrosoftNetCoreAppVersion)</DotNetCliVersion>
+    <DotNetCliRuntime>win-x86</DotNetCliRuntime>
```

Comment:
Is `RuntimeIdentifier` set at this point, can you just use that?

Reply:
i only see `RuntimeIdentifiers` set at this point. although i can refactor some of the common logic out so we should be able to use that here